symfony/webpack-encore-pack/1.0/webpack.config.js

```diff
@@ -5,20 +5,27 @@ Encore
     .setOutputPath('public/build/')
     // the public path used by the web server to access the previous directory
     .setPublicPath('/build')
+    // the path inside the Symfony app, as given to the asset() function
+    .setManifestKeyPrefix('build/')
```

Comment:
Unless you're deploying to a sub-directory or a CDN, this is not something you need. And if you DO set your publicPath to an absolute URL, we actually tell you that you need this exact method. Actually, if you set your publicPath to a subdirectory (e.g. `/subdirectory/build`) Encore also tells you in that case that you need to add this.

So, in theory, the code walks you through when you need this. So I don't want to include it for everyone else. But obviously, you had a confusing experience - so what could we have done to improve that?

Reply:
Not that everyone goes looking for docs, but here are the docs about a sub-directory: https://symfony.com/doc/current/frontend/encore/faq.html#my-app-lives-under-a-subdirectory

Reply:
The experience I had was:
1. Refresh page in subdirectory: asset broken. => Missing public path prefix
2. Rebuild assets => error hinting about setManifestKeyPrefix

So yes, it works. But each step was a small wtf with time spent having to concentrate to understand what was happening (or even just displayed, rtfm ;) ).

Does this added line hurt anything? It would remove one oops if not :)